<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLabels</class>
 <widget class="QWidget" name="FormLabels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QToolButton" name="btnAddChild">
       <property name="toolTip">
        <string>添加子标签</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="btnMoveLeft">
       <property name="toolTip">
        <string>左移标签</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="btnMoveUp">
       <property name="toolTip">
        <string>上移标签</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QToolButton" name="btnMoveRight">
       <property name="toolTip">
        <string>右移标签</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="btnDelete">
       <property name="toolTip">
        <string>删除标签</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="btnMoveDown">
       <property name="toolTip">
        <string>下移标签</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="btnAdd">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTreeView" name="treeView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/addnode.png</normaloff>:/Resources/addnode.png</iconset>
   </property>
   <property name="text">
    <string>添加标签</string>
   </property>
  </action>
  <action name="actionAddChild">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/addchildnode.png</normaloff>:/Resources/addchildnode.png</iconset>
   </property>
   <property name="text">
    <string>添加子标签</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/deletenode.png</normaloff>:/Resources/deletenode.png</iconset>
   </property>
   <property name="text">
    <string>删除标签</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/moveup.png</normaloff>:/Resources/moveup.png</iconset>
   </property>
   <property name="text">
    <string>上移标签</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/movedown.png</normaloff>:/Resources/movedown.png</iconset>
   </property>
   <property name="text">
    <string>下移标签</string>
   </property>
  </action>
  <action name="actionMoveLeft">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/moveleft.png</normaloff>:/Resources/moveleft.png</iconset>
   </property>
   <property name="text">
    <string>左移标签</string>
   </property>
  </action>
  <action name="actionMoveRight">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/moveright.png</normaloff>:/Resources/moveright.png</iconset>
   </property>
   <property name="text">
    <string>右移标签</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
